app/views/govuk_publishing_components/components/docs/single_page_notification_button.yml

```diff
@@ -0,0 +1,32 @@
+name: Single page notification button
+description: A button that subscribes the user to email notifications to a page
+body: |
+  By default, the component displays with the "Get emails about this page" state. The component does not render without the `base_path` parameter.
+
+  The `base_path` is necessary for [checking if an email subscription is active on page load](https://github.com/alphagov/account-api/blob/main/docs/api.md#get-apipersonalisationcheck-email-subscriptiontopic_slug) and the creation/deletion of an email notification subscription.
+
+  When the button is clicked, the `base_path` is submitted to an endpoint which proceeds to check the user's authentication status and whether they are already subscribed to the page or not. Depending on these factors, they will be routed accordingly.
+accessibility_criteria: |
+  - The bell icon must be presentational and ignored by screen readers.
```

Comment:
I think we might need some more accessibility criteria but this is non-blocking for now - I have an idea for this, will catch up.